src/assets/bootstrap/_variables.scss

```
@@ -74,6 +74,7 @@ $warning:       $yellow !default;
 $danger:        $red !default;
 $light:         $white !default;
 $dark:          $gray-800 !default;
+$twitter:       #7FE2EB !default;
```

```suggestion
$twitter:       #59BAFF !default;
```
Acho que essa cor se aproxima mais ao mock!